en/development/sessions.rst

```diff
@@ -383,7 +383,7 @@ Destroying the Session
 
 .. php:method:: destroy()
 
-Destroying the session is useful when users log out. To destroy a session, use
+Destroying the session is useful when users logout. To destroy a session, use
```

Comment:
I think i't still "log out" here, as it is used as a verb and not describing the action or the functionality. But that's nitpicking...